WcaOnRails/lib/dues_calculator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module DuesCalculator
+  def self.update_exchange_rates_if_needed
+    if !Money.default_bank.rates_updated_at || Money.default_bank.rates_updated_at < 1.day.ago
+      Money.default_bank.update_rates
+    end
+  end
+
+  def self.dues_per_competitor_in_usd(country_iso2, base_entry_fee_lowest_denomination, currency_code)
+    country_band = CountryBand.find_by(iso2: country_iso2)&.number
+
+    DuesCalculator.update_exchange_rates_if_needed
+    input_money_us_dollars = Money.new(base_entry_fee_lowest_denomination.to_i, currency_code).exchange_to("USD").amount
```

Comment:
Why are you converting this to an integer?
- If it already is an integer, then `to_i` is not necessary.
- If it already is a money object, then creating a `Money.new` is not necessary.

Reply:
It can be string when it comes from competition_controller, hence added `to_i`.

Reply:
Then `competition_controller` should be responsible for converting it to an integer before passing it to the dues calculator.